### ISSUE TITLE
entity type 0 issue

### ISSUE THREAD

**Issue by theonediepplayer** [Cazka/diepAPI#54]
sometimes the player entity can be found as destroyer branch bullets

**Comment by Cazka** [comment]
Yeah thats because a destroyer bullet is roughly the same size as a player entity. 

A quick fix could be to check if `entity.parent` exists, since that field is only set on bullets.

To make it more robust, we would need to check if the entity has a barrel attached to it. #11

**Comment by weaponcubemap** [comment]
Would be useful if player entity detected as player, not as bullet or etc
